cpp/tests/structs/structs_column_tests.cpp

```diff
@@ -637,7 +637,7 @@ TEST_F(StructColumnWrapperTest, TestStructsColumnWithEmptyChild)
     cudf::test::detail::make_null_mask(mask_vec.begin(), mask_vec.end());
   auto structs_col =
     cudf::make_structs_column(num_rows, std::move(cols), null_count, std::move(null_mask));
-  EXPECT_NO_THROW(structs_col->view());
+  EXPECT_NO_THROW(auto _ = structs_col->view());
```

Comment:
I had to capture the returned value because `view` is marked as nodiscard. Presumably the test is just to make sure that view construction succeeds, is that what you're asking? We could also tell the linter to ignore this one.

Reply:
So this is just testing the `view()` member of `cudf::column` does not throw an exception which likely will not every happen: https://github.com/rapidsai/cudf/blob/9b2f892c5ec59605bfdc3a2abe4885176950589a/cpp/src/column/column.cu#L89
I'm guessing this was meant to check `cudf::make_structs_column` does not throw an exception which mostly definitely can throw:
https://github.com/rapidsai/cudf/blob/9b2f892c5ec59605bfdc3a2abe4885176950589a/cpp/src/structs/structs_column_factories.cu#L31
So I think 
```
  EXPECT_NO_THROW(auto structs_col =
    cudf::make_structs_column(num_rows, std::move(cols), null_count, std::move(null_mask)));
```
would be more appropriate. And remove the check for the `view()` call.

Reply:
Makes sense, I'll do that.